src/functions/Output.ps1

```diff
@@ -309,6 +310,10 @@ function Write-PesterReport {
     )
     # if(-not ($PesterState.Show | Has-Flag Summary)) { return }
 
+    if ($RunResult.Tests.ErrorRecord.FullyQualifiedErrorID -contains 'PesterTestPending') {
+        Write-PesterHostMessage '**DEPRECATED**: The Set-ItResult -Pending parameter is deprecated. It will be removed in a future version of Pester.' -ForegroundColor $ReportTheme.Warning
+    }
+
```

Comment:
I agree, and think it needs a blank line as well to not clutter the "Passed/Failed/Skipped" line. How do you like this?

```PowerShell
Pester v5.5.0

Starting discovery in 1 files.
Discovery found 5 tests in 8ms.
Running tests.
Describing a
  [+] a 6ms (3ms|3ms)
  [-] b 5ms (3ms|1ms)
   Expected $false, but got $true.
   at $true | Should -BeFalse, :10
   at <ScriptBlock>, <No file>:10
  [?] c is inconclusive, because INCONCLUSIVE: demo inconclusive 4ms (2ms|2ms)
  [!] d is pending, because PENDING: demo pending 5ms (3ms|2ms)
  [!] e is skipped, because demo skipped 4ms (2ms|1ms)
Tests completed in 82ms
Tests Passed: 1, Failed: 1, Skipped: 2, Inconclusive: 1, NotRun: 0

**DEPRECATED**: The Set-ItResult -Pending parameter is deprecated. It will be removed in a future version of Pester.
```

Reply:
Slightly reworded the deprecation message, to point out that it is the parameter and not Set-ItResult that is being deprecated.